src/Uno.UI/UI/Xaml/Controls/MenuFlyout/MenuFlyout.cs

```diff
@@ -1,50 +1,52 @@
-﻿using System;
+#pragma warning disable 649
+#pragma warning disable 414 // assigned but its value is never used
+
+using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Collections.Specialized;
+using System.Diagnostics;
 using System.Linq;
 using System.Text;
 using Uno.Extensions;
+using Uno.UI;
+using Windows.Foundation;
 using Windows.Foundation.Collections;
+using Windows.Networking.Sockets;
+using Windows.UI.Xaml.Automation.Peers;
 using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
 using Windows.UI.Xaml.Markup;
+using Windows.UI.Xaml.Media.Animation;
 
 namespace Windows.UI.Xaml.Controls
 {
 	[ContentProperty(Name = "Items")]
-	public partial class MenuFlyout : FlyoutBase
-    {
-		private MenuFlyoutPresenter _presenter;
+	public partial class MenuFlyout : FlyoutBase, IMenu
+	{
+		private readonly ObservableVector<MenuFlyoutItemBase> m_tpItems;
 
-		public MenuFlyout()
-		{
-			var collection = new ObservableVector<MenuFlyoutItemBase>();
-			collection.VectorChanged += OnItemsVectorChanged;
+		// In Threshold, MenuFlyout uses the MenuPopupThemeTransition.
+		// UNO TODO private Transition m_tpMenuPopupThemeTransition = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5205492)